scipy/spatial/qhull.pyx

```diff
@@ -2290,6 +2300,19 @@ class ConvexHull(_QhullUser):
         triangulation due to numerical precision issues.
 
         If option "Qc" is not specified, this list is not computed.
+    good : A one-dimensional Boolean array indicating which facets are
```

Comment:
This line should be adjusted to more closely match our usual standard. See the issue in [the circle ci render](https://11925-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.spatial.ConvexHull.html#scipy.spatial.ConvexHull) at time of writing.

Beyond that, the `ConvexHull` visibility additions look pretty nice there, but I'm biased.

Reply:
I've tried adjusting this -- we'll see if it looks slightly better once the CI docs are built